CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * Fixed a potential cause for Realm file corruptions (never reported).
 * Add `@Override` annotation to proxy class accessors and stop using raw type in proxy classes in order to remove warnings from javac (#4329).
 * `findFirstAsync()` now returns an invalid object if there is no object matches the query condition instead of running the query repeatedly until it can find one (#4352).
+* Fixed a bug with setting sync log level (#4337).
```

Comment:
I kinda prefer the [realm-cocoa changelog](https://github.com/realm/realm-cocoa/blob/master/CHANGELOG.md) where they start every line with `Fix`, because that way the verb isn't omitted.

Probably a personal thing.